### ISSUE TITLE
Add copy button to messages

### ISSUE THREAD

**Issue by rodber** [chevere/xr#32]
Add a copy button which copy the message to the clipboard.

**Comment by rodber** [comment]
<img width="815" alt="image" src="https://user-images.githubusercontent.com/20590102/152643276-ff6e6e20-63e9-425c-b195-53aed343317e.png">

^ It copies this to the clipboard:

```plain
Arg:inspiration Chevere\Message\Message#11
 private $trTable array (size=1)
   %package% => array (size=2)
      0 => string code (length=4)
      1 => string spatie/ray (length=10)
 private $template string Did you heard about %package%? (length=30)
------------------------------------------------------------
#0 /Users/rodolfo/git/chevere/xr/demo.php:42
xr(object Chevere\Message\Message#11, string hello-world, string 😎, integer 1)
------------------------------------------------------------
10:30:43・demo.php:42
```

**Comment by rodber** [comment]
closed in f4c535a1d2e971e22fc5971933aa803e66a57525
